sites/fast-creator/app/components/wrapper/wrapper.styles.ts

```diff
@@ -0,0 +1,88 @@
+import { css } from "@microsoft/fast-element";
+
+const topLeft: string = ".tick:nth-child(1)";
+const topRight: string = ".tick:nth-child(2)";
+const rightTop: string = ".tick:nth-child(3)";
+const rightBottom: string = ".tick:nth-child(4)";
+const bottomRight: string = ".tick:nth-child(5)";
+const bottomLeft: string = ".tick:nth-child(6)";
+const leftBottom: string = ".tick:nth-child(7)";
+const leftTop: string = ".tick:nth-child(8)";
+
+export const WrapperStyles = css`
+    :host {
+    }
+
+    .control {
+        position: fixed;
+        border: 1px solid #FB356D;
+        box-sizing: border-box;
+        pointer-events: none;
+        opacity: 0;
+    }
+
+    .preselect {
+        opacity: 0.2;
+    }
+
+    .control.preselect,
+    .control.active {
+        opacity: 1;
+    }
+
+    .tick {
+        position: absolute;
+        width: 0;
+        height: 8px;
+        border-left: 1px solid #FB356D;
+        box-sizing: border-box;
+    }
+
+    ${rightTop},
+    ${rightBottom},
+    ${leftBottom},
+    ${leftTop} {
+        transform-origin: left top;
```

Comment:
what happens with RTL or LTR scenarios?

Reply:
it's not relevant here, this will work in either scenario since these values are mirrored.